Add noisy circuit example #1782

iamvamsikrishnad opened the pull request.

dabacon commented on examples/noisy_simulation_example.py:
```diff
@@ -0,0 +1,43 @@
+                          noise=cirq.ConstantQubitNoiseModel(
+                              cirq.amplitude_damp(amplitude_damp)),
+                          repetitions=100)
+    print("Noise model: ConstantQubitNoiseModel with",
```
make this more decriptive like
"ConstantQubitNoiseModel with amplitude damping of rate"

dabacon commented on examples/noisy_simulation_example.py:
```diff
@@ -0,0 +1,43 @@
+
+
+def main():
+    print("Iteration 1:")
```
Make a list of the amplitudes you are damping by and then just iterate over enumerate(your list) so you don't have to repeat all of this.

Strilanc commented on examples/noisy_simulation_example.py:
```diff
@@ -0,0 +1,36 @@
+    """Demonstrates a noisy circuit simulation.
+    """
+    q = cirq.NamedQubit('q')
+    noisy_circuit = cirq.Circuit.from_ops(
```
Just `circuit`. The circuit itself is a noiseless spec.

iamvamsikrishnad replied:
Done.

iamvamsikrishnad commented:
@Strilanc : I have addressed your comment, but still I see an issue (Merging can be performed automatically once the requested changes are addressed.) in merging the changes into master. Can you help me with this.
